pkg/morph/client/reload.go

```diff
@@ -0,0 +1,25 @@
+package client
+
+import "slices"
+
+// Reload allows runtime reconfiguration for WithEndpoints parameter.
+func (c *Client) Reload(opts ...Option) {
+	cfg := new(cfg)
+	for _, o := range opts {
+		o(cfg)
+	}
+
+	c.cfg.endpoints = cfg.endpoints
+
+	c.endpoints = cfg.endpoints
+
+	var conn = c.conn.Load()
+	if conn == nil {
+		return
+	}
+	currentEndpointID := conn.client.Endpoint()
+
+	if slices.Index(c.endpoints, currentEndpointID) == -1 {
+		conn.client.Close()
```

Comment:
Brutal. I'd rather document that we're keeping current connection.

Reply:
Added to a `Reload` function description.

Reply:
I mean dropping this code, RPC reconnections are costly for node, so we can avoid them and use current node while it's alive.

Reply:
@roman-khimov, this happens only if we are connected to a node that should not be used anymore according to the updated nodes list

Reply:
My intention is not to drop this connection anyway. And I'm not sure you can rely on these comparisons.

Reply:
Ah, now I get it. That is, even if the new config does not have this endpoint, keep in touch with it?

Reply:
Yes.

Reply:
you are connected to node X, then you updated to a list that does not have node X, what is your expectation as an admin?

Reply:
Depends on documentation.

Reply:
I'm also for switching, it seems logical to me. But if this behavior is documented, then fine.

Reply:
Let's ask @532910.